hack/build/common.sh

```diff
@@ -27,3 +47,5 @@ CACHE_DIR=${OUT_DIR}/gocache
 VENDOR_DIR=${CDI_DIR}/vendor
 ARCHITECTURE="${BUILD_ARCH:-$(uname -m)}"
 HOST_ARCHITECTURE="$(uname -m)"
+CDI_CRI="$(determine_cri_bin)"
+CDI_CONTAINER_BUILDCMD="$(determine_container_buildcmd)"
```

Comment:
maybe if the selected CRI is podman, enforce to have buildah ? else it is a mix between podman and docker

Reply:
Done